Traceback (most recent call last):
  File "/home/xethrus/paidProject/AvaliablilityProgram/status_expiration_task.py", line 17, in <module>
    schedule.run_pending()
  File "/home/xethrus/.local/lib/python3.10/site-packages/schedule/__init__.py", line 780, in run_pending
    default_scheduler.run_pending()
  File "/home/xethrus/.local/lib/python3.10/site-packages/schedule/__init__.py", line 100, in run_pending
    self._run_job(job)
  File "/home/xethrus/.local/lib/python3.10/site-packages/schedule/__init__.py", line 172, in _run_job
    ret = job.run()
  File "/home/xethrus/.local/lib/python3.10/site-packages/schedule/__init__.py", line 661, in run
    ret = self.job_func()
  File "/home/xethrus/paidProject/AvaliablilityProgram/status_expiration_task.py", line 12, in status_expiration
    if expiration_time <= datetime.datetime.now():
NameError: name 'expiration_time' is not defined
